src/js/locale/fr.yaml

```diff
@@ -37,6 +42,10 @@ actions:
   confirm: Êtes-vous sûr?
   delete: Supprimer
   remove: Retirer
+  cancel: Annuler
+  created: %{name} créé
+  saved: %{name} sauvegardé
+  deleted: %{name} supprimé

```

Comment:
When using variables, they need to be wrapped in single quotes (`'`). These lines should be:
```
  created: '%{name} créé'
  saved: '%{name} sauvegardé'
  deleted: '%{name} supprimé'
```